### ISSUE TITLE
Rx300 Last temp problem?

### ISSUE THREAD

**Issue by Poopcheeks** [maelstrom2001/ArcticFox#309]
When I'm using my troll RTA my last temperature stays at 70F but then i use my pharaoh RTA it doesn't do it. Sometimes my troll works sometimes it doesn't, does anyone know why this is happening?

**Comment by TBXin** [comment]
Hello,

Last Temp means last temperature of coil that was gained in the TC mode. If you are running your attys in the VW mode Last Temp will be always 70F / 21С.

**Comment by Poopcheeks** [comment]
Yeah i am in TC Mode i use both tanks in TC mode, But i get that glitch or something on my Troll RTA
